tests/src/main/resources/application.conf

```diff
@@ -0,0 +1,22 @@
+akka {
+  test {
+    timefactor = 8.0
+  }
+
+  http {
+    server.parsing.max-content-length = 10g
+    host-connection-pool  {
+      max-connections   = 16
+      max-open-requests = 64
+    }
+  }
+
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+  logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
+  log-dead-letters = off
+  stdout-loglevel = "OFF"
+  log-dead-letters-during-shutdown = off
+  loglevel = INFO
```

Comment:
Do we really want the integration tests at this loglevel by default? WARN or ERROR would be better I guess